docs/07_Installing_Essential_Cluster_Services.md

```diff
@@ -139,10 +141,11 @@ EOF
 Enable NFS with:
 
 ```bash
-sudo nfsd enable
+sudo systemctl enable nfs-kernel-server.service  # if its not already
+sudo systemctl start nfs-kernel-server.service  
 ```
 
-(or use `sudo nfsd update` if `nfsd` is already enabled)
+(or use `sudo systemctl restart nfs-kernel-server.service` if `nfs-kernel-server` is already enabled)
```

Comment:
It should be enough to simply call `exportfs -a` (this is what the script does).
I think I just forgot to update this when I ported the guide from macOS to Linux.